compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -721,6 +730,13 @@ prettyNodeNode nn = do
               , prettyContractId node_ExerciseTargetContractId
               , parens (prettyMay "<missing TemplateId>" (prettyDefName world) node_ExerciseTemplateId)
               ]
+           $$ foldMap
+              (\key ->
+                let prettyKey = prettyMay "<KEY?>" (prettyValue' False 0 world) $ keyWithMaintainersKey key
+                in
+                hsep [ keyword_ "using key", prettyKey ]
```

Comment:
consider using `by key` instead of `using key` to more closely match the transaction structure and the name of the tufncitons in Daml.

Reply:
Changed `using key` to `by key`